site/docs/types/1/0115-teams.md

```diff
@@ -0,0 +1,12 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the Egeria project. -->
+
+# 0115 Teams
+
+A team is a group of people who are working to a common goal. The [ActorProfile](0110-actors.md) is extended to capture more information about a team. This is recorded in the Team entity.
```

Comment:
- Should have a `## Team` heading
- ActorProfile should be back-ticked as `ActorProfile` and linked directly to that heading in the markdown file (e.g. `/egeria-docs/types/1/0110-actors/#actorprofile`)
- `This is recorded in the Team entity` should be `This is recorded in the *Team* entity`.